tests/tracing/test_dnatracing_multigrain.py

```diff
@@ -301,7 +323,7 @@ def test_trace_mask(
                 {
                     "molecule_number": [0, 1],
                     "image": ["multigrain_topostats", "multigrain_topostats"],
-                    "contour_length": [1.2382864476914832e-07, 7.617314045334366e-08],
+                    "contour_length": [5.684734982126663e-08, 7.574136072208753e-08],
```

Comment:
Grain lengths look like they are roughly/broadly halving but the image examples don't look like that for most grains. Just wanted to check that this reduction in length is correct.

Reply:
Thank you for being so vigilant! I believe this is correct. I saved the splined trace for that molecule in main and my branch, and this is the difference in the splines:

<img width="664" alt="image" src="https://github.com/AFM-SPM/TopoStats/assets/86117496/2e884665-6dd8-4b3b-895c-b994d3505fd8">

So the spline is much shorter now since it is not jumping about all over the place, artificially increasing the length.

Does this seem okay to you?

In my investigations, I have found that contour length is possibly being calculated badly, and providing systematically incorrect results, but that's for more investigation and a separate issue focusing on measure_contour_length.